libsrc/telestrat/clrscr.s

```diff
@@ -34,5 +34,14 @@
     sta     SCRY
     lda     #$00
     sta     SCRX
+
+    lda     #$00
```

Comment:
If you use an index register, then you can put `$01` into `SCRY`, decrement it, then put zero into the other three variables.

Reply:
Done